cmd/log_drains.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"github.com/Scalingo/cli/appdetect"
+	"github.com/Scalingo/cli/log_drains"
+	"github.com/urfave/cli"
+)
+
+var (
+	LogDrainsListCommand = cli.Command{
+		Name:     "log-drains",
+		Category: "Log drains",
+		Flags:    []cli.Flag{appFlag},
+		Usage:    "List the log drains of an application",
+		Description: `List all the log drains of an application:
+
+    $ scalingo --app my-app log-drains
+
+    # See also commands 'log-drains-add', 'log-drains-remove' and 'log-drains-info'`,
+
+		Action: func(c *cli.Context) {
+			currentApp := appdetect.CurrentApp(c)
+			var err error
+			if len(c.Args()) == 0 {
+				err = log_drains.List(currentApp)
+			} else {
+				cli.ShowCommandHelp(c, "log-drains")
```

Comment:
Please get rid of this case as early as possible. Hence the first thing to do in this function is:

```go
			if len(c.Args()) != 0 {
				cli.ShowCommandHelp(c, "log-drains")
				return
			}
```